Repositories/Contents.md

```diff
@@ -65,3 +66,106 @@ GET /repos/:username/:reponame/archive/:ref:format
 Status: 200 OK
 Content-Type: application/octet-stream
 ```
+
+## Get Contents
+
+```
+GET /repos/:username/:reponame/contents/:ref/:path
+```
+
+### Response if content is a file
+```
+Status: 200 OK
+```
+```json
+{
+  "type": "file",
+  "size": 0,
+  "name": "text1.text",
+  "path": "text1.text",
+  "sha": "e69de29bb2d1d6434b8b29ae775ad8c2e48c5391",
+  "url": "https://gogs.io/api/v1/repos/root/test-symlink/contents/text1.text",
```

Comment:
Up until now, we use `https://try.gogs.io` for example URLs.

Reply:
fixed.